src/Symfony/Component/ExpressionLanguage/CHANGELOG.md

```diff
@@ -8,11 +8,7 @@ CHANGELOG
  * Deprecate loose comparisons when using the "in" operator; normalize the array parameter
    so it only has the expected types or implement loose matching in your own expression function
 
-6.2
```

Comment:
The changes here should be reverted, right?

Reply:
Yes, I will this weekend. Thank so much it must be an error rebasing the branch :+1:

Reply:
@xabbuh DONE :)